base/binaryplatforms.jl

```diff
@@ -170,20 +170,18 @@ end
 
 
 # Allow us to easily serialize Platform objects
-function Base.repr(p::Platform; context=nothing)
-    str = string(
-        "Platform(",
-        repr(arch(p)),
-        ", ",
-        repr(os(p)),
-        "; ",
-        join(("$(k) = $(repr(v))" for (k, v) in tags(p) if k ∉ ("arch", "os")), ", "),
-        ")",
-    )
+function Base.show(io::IO, p::Platform; context=nothing)
```

Comment:
```suggestion
function Base.show(io::IO, p::Platform)
```
In `repr`, this is for passing through an IOContext, but in `show` that's in the `io` argument.